magda-typescript-common/src/typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.json" {
```

Comment:
I think there's an option in tsconfig that does this for you now?

Reply:
👍 Removed the `typings.d.ts`